changelog.d/290.feature.rst

```diff
@@ -0,0 +1,11 @@
+Create semver 3.0.0-alpha0
```

Comment:
How does towncrier know that the number here is a reference to the PR #290 instead of the issue 290?

Reply:
Well, that could be a problem. At the moment, I haven't a good solution for that to be honest.

As far as I've understood it, you can configure Towncrier to some degree (within the  `pyproject.toml` config). Additionally there is a template (`changelog.d/_template.rst`) that renders the output. I haven't dug deeper if it is possible to distinguish between issue and pull request.

I thought of that as a minor issue for the time being, but certainly we should solve that.

Reply:
Oh wait, I solved that already! Ha!

If you want to reference to a pull request only, use the following command:

```
$ tox -e changelog -- create pr1000.feature
```

That will generate `changelog.d/pr1000.feature.rst`. After you have added your content to this file, it renders like this:

```rst
Feature
----------
* :pr:`100`: This is the content of our pull request 100
```

So it's possible.